website/docs/r/cen_instance_attachment.html.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: "alicloud"
+page_title: "Alicloud: alicloud_cen_instance_attachment"
+sidebar_current: "docs-alicloud-cen-instance-attachment"
+description: |-
+  Provides a Alicloud CEN child instance attachment resource.
+---
+
+# alicloud\_cen_instance_attachment
+
+Provides a CEN child instance attachment resource.
+
+## Example Usage
+
+Basic Usage
+
+```
+# Create a new instance-attachment and use it to attach one child instance to a new CEN
+variable "name"{
+	default = "tf-testAccCenInstanceAttachmentBasic"
+}
+
+resource "alicloud_cen_instance" "cen" {
+	name = "${var.name}"
+	description = "terraform01"
+}
+
+resource "alicloud_vpc" "vpc" {
+	name = "${var.name}"
+	cidr_block = "192.168.0.0/16"
+}
+
+resource "alicloud_cen_instance_attachment" "foo" {
+    instance_id = "${alicloud_cen_instance.cen.id}"
+	child_instance_id = "${alicloud_vpc.vpc.id}"
+	child_instance_region_id = "cn-beijing"
+}
+```
+## Argument Reference
+
+The following arguments are supported:
+
+* `cen_id` - (Required) The ID of the CEN.
+* `child_instance_id` - (Required) The ID of the child instance to attach.
+* `child_instance_region_id` - (Required) The region ID of the child instance to attach.
+
+~>**NOTE:** Ensure that the child instance is not used in Express Connect.
+
+## Attributes Reference
+
+The following attributes are exported:
+
```

Comment:
I think the exported parameters missing `id`.

In addition, if all of parameters are inputting, them can be ignored in the `exported block`.